app/views/admin/content/_form.html.erb

```diff
@@ -79,23 +80,23 @@
                 </p>
               </div>
             </div>
-            <div class="control-group">
-              <p>
-                <%= t('.status') %>: <strong><%= @article.state.to_s.downcase %></strong>
-                <%= toggle_element('status') %>
-              </p>
-              <div id="status" class="collapse">
-                <label for="article_published" class="checkbox">
-                  <%= check_box 'article', 'published?' %>
-                  <%= t('.published') %>
-                </label>
-                <p>
-                  <span class="btn btn-mini btn-default">
-                    <%= toggle_element('status', t('.ok')) %>
-                  </span>
-                </p>
-              </div>
-            </div>
+<!--            <div class="control-group">-->
```

Comment:
For code we do need to delete, just remove it and we can look in the diff later.